2024-08-29 00:15:51,195 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Current SDK version is 0.17.8
2024-08-29 00:15:51,196 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Configure stats pid to 1815938
2024-08-29 00:15:51,196 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Loading settings from /home/ray/.config/wandb/settings
2024-08-29 00:15:51,197 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Loading settings from /home/ray/mnt/cluster_storage/ai_geolocation/geo-clip/geoclip/wandb/settings
2024-08-29 00:15:51,197 WARNING MainThread:1815938 [wandb_setup.py:_flush():77] Unknown environment variable: WANDB_POPULATE_RUN_LOCATION_HOOK
2024-08-29 00:15:51,197 WARNING MainThread:1815938 [wandb_setup.py:_flush():77] Unknown environment variable: WANDB_SETUP_API_KEY_HOOK
2024-08-29 00:15:51,198 WARNING MainThread:1815938 [wandb_setup.py:_flush():77] Unknown environment variable: WANDB_PROCESS_RUN_INFO_HOOK
2024-08-29 00:15:51,198 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Loading settings from environment variables: {}
2024-08-29 00:15:51,198 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Applying setup settings: {'_disable_service': False}
2024-08-29 00:15:51,199 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-08-29 00:15:51,199 INFO    MainThread:1815938 [wandb_setup.py:_flush():77] Applying login settings: {}
2024-08-29 00:15:51,200 INFO    MainThread:1815938 [wandb_init.py:_log_setup():524] Logging user logs to /home/ray/mnt/cluster_storage/ai_geolocation/geo-clip/geoclip/wandb/run-20240829_001551-dl1g9fmx/logs/debug.log
2024-08-29 00:15:51,200 INFO    MainThread:1815938 [wandb_init.py:_log_setup():525] Logging internal logs to /home/ray/mnt/cluster_storage/ai_geolocation/geo-clip/geoclip/wandb/run-20240829_001551-dl1g9fmx/logs/debug-internal.log
2024-08-29 00:15:51,200 INFO    MainThread:1815938 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7e91331fb490>
2024-08-29 00:15:51,201 INFO    MainThread:1815938 [wandb_init.py:init():607] calling init triggers
2024-08-29 00:15:51,201 INFO    MainThread:1815938 [wandb_init.py:init():614] wandb.init called with sweep_config: {}
config: {'learning_rate': {0.0001}, 'step_size': {30}, 'epochs': {10}, 'optimizer_gamma': {0.1}}
2024-08-29 00:15:51,201 INFO    MainThread:1815938 [wandb_init.py:init():657] starting backend
2024-08-29 00:15:51,202 INFO    MainThread:1815938 [wandb_init.py:init():661] setting up manager
2024-08-29 00:15:51,206 INFO    MainThread:1815938 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-08-29 00:15:51,207 INFO    MainThread:1815938 [wandb_init.py:init():669] backend started and connected
2024-08-29 00:15:51,209 INFO    MainThread:1815938 [wandb_run.py:_label_probe_notebook():1344] probe notebook
2024-08-29 00:15:51,210 INFO    MainThread:1815938 [wandb_run.py:_label_probe_notebook():1354] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-08-29 00:15:51,210 INFO    MainThread:1815938 [wandb_init.py:init():767] updated telemetry
2024-08-29 00:15:51,222 INFO    MainThread:1815938 [wandb_init.py:init():800] communicating run to backend with 90.0 second timeout
2024-08-29 00:15:51,450 INFO    MainThread:1815938 [wandb_init.py:init():851] starting run threads in backend
2024-08-29 00:15:55,449 INFO    MainThread:1815938 [wandb_run.py:_console_start():2463] atexit reg
2024-08-29 00:15:55,450 INFO    MainThread:1815938 [wandb_run.py:_redirect():2309] redirect: wrap_raw
2024-08-29 00:15:55,451 INFO    MainThread:1815938 [wandb_run.py:_redirect():2374] Wrapping output streams.
2024-08-29 00:15:55,452 INFO    MainThread:1815938 [wandb_run.py:_redirect():2399] Redirects installed.
2024-08-29 00:15:55,454 INFO    MainThread:1815938 [wandb_init.py:init():894] run started, returning control to user process
2024-08-29 00:15:55,457 INFO    MainThread:1815938 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-08-29 00:15:55,457 INFO    MainThread:1815938 [wandb_init.py:_pause_backend():435] pausing backend
2024-08-29 00:16:02,041 INFO    MainThread:1815938 [wandb_init.py:_resume_backend():440] resuming backend
2024-08-29 00:17:56,385 INFO    MainThread:1815938 [jupyter.py:save_ipynb():372] not saving jupyter notebook
2024-08-29 00:17:56,385 INFO    MainThread:1815938 [wandb_init.py:_pause_backend():435] pausing backend
